builder-gradle4/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM jenkinsxio/builder-base:1
+
+CMD ["gradle"]
+
+ENV GRADLE_HOME /opt/gradle
+ENV GRADLE_VERSION 4.10.3
+
+RUN set -o errexit -o nounset \
+    && echo "Set URLS" \
+    && GRADLE_URL="https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip" \
+    && SHA256_URL="${GRADLE_URL}.sha256" \
+    \
+    && echo "Determine sha256sum" \
+    && GRADLE_DOWNLOAD_SHA256=$(curl -sLf "${SHA256_URL}") \
```

Comment:
Hi @garethjevans, jFYI - I've removed the hard-coded sha256 and taken it from the website instead. Hope that's okay.